api/v1/clusterextensionrevision_types.go

```diff
@@ -22,7 +22,24 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-const ClusterExtensionRevisionKind = "ClusterExtensionRevision"
+const (
+	ClusterExtensionRevisionKind = "ClusterExtensionRevision"
+
+	// Condition Types
+	TypeAvailable = "Available"
+	TypeSucceeded = "Succeeded"
+
+	// Condition Reasons
+	ReasonObjectAvailable           = "Available"
```

Comment:
I think we should stick with the pattern `Reason<Reason> = "<Reason>"`
```suggestion
	ReasonAvailable           = "Available"
```

Reply:
`ReasonAvailable` is already set in `clustercatalog_types.go`, which was why initially I moved that one to the `common_types.go` to share it with CER. So we either need to combine them like I did before or have another distinct name, though it doesn't have to be `ReasonObjectAvailable`, of course 😃

Reply:
...although following your recommendation [here](https://github.com/thetechnick/operator-controller/pull/10#discussion_r2320576333), this will resolve itself if I name it `ClusterExtensionRevisionReasonAvailable`.